Examples/Python/src/Geometry.cpp

```diff
@@ -22,9 +22,21 @@ using namespace pybind11::literals;
 namespace Acts::Python {
 void addGeometry(Context& ctx) {
   auto m = ctx.get("main");
-
-  py::class_<Acts::Surface, std::shared_ptr<Acts::Surface>>(m, "Surfaces");
-
+  {
+    py::class_<Acts::Surface, std::shared_ptr<Acts::Surface>>(m, "Surfaces")
```

Comment:
I guess if we're already doing this, I'd add the extra level and make bindings for `GeometryIdentifier` and then return that from this the `geometryId` method also in python.